lib/routes.js

```diff
@@ -24,8 +24,14 @@ router.addRoute('/chain', function(req, res, params, splats) {
 router.addRoute('/boot/:macaddr', function boot(req, res, params, splats) {
   console.log("Got a BOOT request");
   console.log("Just got word that "+params.params.macaddr+" just booted");
-
-  var stat = fs.statSync(__dirname + '/static/' + params.params.macaddr);
+  
+  try {
```

Comment:
oh bah... right, since it's a sync call. should probably change that to an async one.